core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt

```diff
@@ -469,12 +469,12 @@ fun LpPaymentsRentCard(
             )
             Column(modifier = Modifier.padding(start = LeasePertTheme.sizes.midLargeSize)) {
                 Text(
-                    text = "$$quantity",
+                    text = stringResource(id = R.string.plus_symbol_for_quantities) + "$quantityToPay",
```

Comment:
we need to concatenate all the value in the string from the strings file please

Reply:
Done